### DISCUSSION TITLE
Make a "Guild" Feature which is basically like a group but way more versatile.

### DISCUSSION THREAD

**Discussion by Cynamicial** [stoatchat/discussions#361] [Feature Suggestions]
I think this is a good addition because it will help make more versatile friend groups and maybe even clans.

**Comment by wait-what** [comment]
Revolt has servers though?

**Reply by ChrisGergler** [reply]
Sub-groups like those in Guilded have been one of the most useful tools I've found. It's easier on the user to create groups within a server than it is to create a whole new server.

There's also community Utility in this.

- Collab spaces for specified groups
- Doing specific games or factions within a community without requiring the oversight of another server stack.
- TTRPG Groupings for living world games

Generally speaking, I'm anti-clutter but having groups on guilded was one of the main attractions as an alternative to Discord for me.

**Comment by amytimed** [comment]
fun fact: on discord, servers are called guilds in the API

the reason is because thats what they are, same for revolt

**Reply by insertish** [reply]
We don't refer to servers as guilds, they're just servers in both the API and UI.

**Reply by amytimed** [reply]
yes, but my point is that revolt servers are derived from discord servers right? and discord servers are guilds
